web/src/app/components/timeline/timeline.component.ts

```diff
@@ -45,38 +54,23 @@ export class TimelineComponent implements AfterViewInit {
       new HumanizeDurationLanguage();
   private static readonly HUMANIZER: HumanizeDuration =
       new HumanizeDuration(TimelineComponent.LANG_SERVICE);
-  private static readonly COLOR_LIGHT_GRAY: string = '#d3d3d3';
+  private static readonly FIVE_MINUTES: number = 5 * 60 * 60 * (10 ** 3);
```

Comment:
I think `1000` is easier to read than `(10 ** 3)`.

It might also be better to name this constant based on what is used for.